### PR TITLE
Allow the ability to easily remove primary keys

### LINKED ISSUE #1742: Allow the ability to easily remove primary keys
### Problem Description

SDV version: 1.9.0
Currently, the metadata auto-detection assigns the primary key to a column that could be a good candidate for it. However because this is not always accurate, the user should be able to remove the primary key from the metadata.

### API

**Single Table**: For `SingleTableMetadata`, add a `remove_primary_key` method:
```python
metadata.remove_primary_key()
```

This method doesn't need parameters and remove the primary key from the metadata.

Note that if there was no primary key to begin with then warn the user.
```
Warning: No primary key exists to remove.
```

**Multi Table**: For `MultiTableMetadata` require the `table_name`.
```python
metadata.remove_primary_key(table_name='table_1')
```
If there are any relationships that use the primary key (either as a parent or a child), then remove those relationships and log this.

```
INFO Relationship between 'table_1' and 'table_2' removed because the primary key for 'table_1' was removed.
```

Continue to show a warning it the table does not have a primary key.
```
Warning: No primary key exists to remove.
```

If the table name invalid, show an error
```
Error: Table 'table_1' not found. Please select a table name that exists in the metadata.
```

### PR DESCRIPTION
**Pull request by frances-h**
Resolve #1742 
CU-86az1ez77

### CODE REVIEW

**Comment by sdv-team** [comment]
Task linked: [CU-86az1ez77 SDV - Add remove_primary_key method #1742](https://app.clickup.com/t/86az1ez77)

File: sdv/metadata/multi_table.py [hunk 1]
```diff
@@ -291,6 +291,33 @@ def remove_relationship(self, parent_table_name, child_table_name):
...
+            parent_table = relationship['parent_table_name']
+            child_table = relationship['child_table_name']
+            foreign_key = relationship['child_foreign_key']
+            if ((child_table == table_name and foreign_key == primary_key) or
+                    parent_table == table_name):
+                relationships_to_remove.append(relationship)
+
+        for relationship in relationships_to_remove:
+            self.relationships.remove(relationship)
+
```
[the remark this answers is not in the document]
Comment (amontanez24) [reply sdv/metadata/multi_table.py:320]: After speaking to @npatki, she said that we don't have to error in this method. The user can validate themselves and add the relationships correctly

**Review by amontanez24 — APPROVED** [review]
LGTM!

**Review by R-Palazzo — APPROVED** [review]
LGTM!

**Pull request merged**